tests/integ/modin/test_qcut.py

```diff
@@ -260,3 +260,52 @@ def test_qcut_invalid_quantiles_negative(q):
 
     with pytest.raises(expected_exception, match=expected_message):
         snow_s.quantile(q)
+
+
+@sql_count_checker(
+    query_count=5,
+    join_count=34,
+    union_count=90,
+    high_count_expected=True,
```

Comment:
forgot to mention in my review -- should we add `skipif(running_on_public_ci())`?

Reply:
yes, that's actually a good idea. There's seems to be right now a problem with the test data setup across the azure-ubuntu and windows deployment. Let me add a general skip. I pinged @sfc-gh-azhan and @sfc-gh-lmukhopadhyay  about it.